app/build.gradle

```diff
@@ -42,6 +42,9 @@ android {
         jvmTarget = '1.8'
     }
 
+    testOptions {
+        unitTests.returnDefaultValues = true
```

Comment:
Is this really mandatory? Android docs seems to tell it should really be avoided, moreover I don't see any strange tests that could justify it

Reply:
I had to because the unit tests won't pass without it

Reply:
i checked on android dev https://developer.android.com/training/testing/unit-testing/local-unit-tests.html